src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,38 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  constructor() {
+    super();
+    this.hidden = false;
+  }
```

Comment:
the Herbivore is missing a parameter 'name' that is expected by the Animal constructor and in the Herbivore class constructor, you must apply the parameter name to the constructor of the parent Animal class using the super(name) methods :)